L/libcgal_julia/build_tarballs.jl

```diff
@@ -3,7 +3,7 @@
 using BinaryBuilder, Pkg
 
 # reminder: change the above version if restricting the supported julia versions
-julia_versions = [v"1.6.3", v"1.7", v"1.8", v"1.9", v"1.10"]
+julia_versions = [v"1.6.3", v"1.7", v"1.8", v"1.9"]
```

Comment:
This can be done later.... For me the priority is julia 1.9.0 for most platforms. After it makes sense to add PRs for stuffs with experimental version of julia (like versio 1.10 and 1.11). But from now, what you had was no more working because precisely stuffs are experimental.